orchestrator/session.go

```diff
@@ -337,6 +350,26 @@ func (s *Session) SendPhonons(keyIndices []uint16) error {
 	return nil
 }
 
+func (s *Session) PostPhonons(pubkey []byte, nonce uint64, keyIndices []uint16) (transferPhononPackets []byte, err error) {
```

Comment:
I'd change the argument name 'pubkey' to 'recipientPubKey' for clarity. It gets confusing with so many different public keys being used throughout the system.